### DISCUSSION TITLE
Participation in different teams

### DISCUSSION THREAD

**Discussion by ProofOfCode08** [aptos-labs/aptos-developer-discussions#493] [Questions] [answered]
### Describe your question in detail.

Hello can a participant be part of multiple teams during this hackathon ?

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by gregnazario** [comment, marked as answer]
There's no rule specifically against this in the code collision hackathon, so it would be allowed
